README.md

```diff
@@ -6,3 +6,47 @@ It's divided into modules, each one having its own README:
 
 * [Todolist Goof](todolist-goof/README.md)
 * [Log4Shell Goof](log4shell-goof/README.md)
+
+---
+## Kubernetes based Todolist + Log4Shell exploit
```

Comment:
@ericsmalling - should this be in the README within the todolist-goof folder?

Reply:
It straddles both - it's a demo of todolist + log4shell

Reply:
I could move it into it's own README though

Reply:
I did that move in in latest commit